NEWS.md

```diff
@@ -32,13 +32,34 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
   intelmq.bots.experts.tor_nodes.expert --update-database
   ```
   The provided shell scripts use these new commands, however they are now deprecated and will be removed in version 3.0.
+- `intelmqctl list queues` is now able to count events in a queue with the `--sum`, `--count` or simple `-s` flag.
```

Comment:
The change is not that `intelmqctl list queues` can "count events in a queue" (it had that functionality already), but that it can give a global sum over all queues.